app/scenes/Etheroscope/components/ContractViewer.js

```diff
@@ -18,8 +18,8 @@ class ContractViewer extends React.Component {
   }
 
   fetchVariableHistory(varName) {
-    var url = '/contracts/' + this.props.contract.address + '/history?variable=' + varName;
-    return fetchJson(url);
+    const url = `/contracts/${this.props.contractAddress}/history?variable=${varName}`
+    return fetchJson(url)
```

Comment:
It should be `this.props.contract.address`

Reply:
Yes my bad I'll change it